### DISCUSSION TITLE
Add switch to enable TLS - LetsEncrypt

### DISCUSSION THREAD

**Discussion by Cieptak** [d3vilh/openvpn-ui#84] [Q&A]
Hello, I have set up the OpenVPN-UI service on my own domain. I would like to make a secure connection to it using an SSL certificate from Let's Encrypt. How to do it safely so as not to lose the current VPN server configuration?

**Comment by Cieptak** [comment]
Maybe it's worth adding a switch to enable automatic Lets Encrypt generation in one of the next updates?

**Reply by ericraskin** [reply]
+1
